packages/core/strapi/src/cli/commands/admin/list-user.ts

```diff
@@ -0,0 +1,61 @@
+import { createCommand } from 'commander';
+import CLITable from 'cli-table3';
+import chalk from 'chalk';
+import { createStrapi, compileStrapi } from '@strapi/core';
+
+import { runAction } from '../../utils/helpers';
+import type { StrapiCommand } from '../../types';
+
+/**
+ * List admin users
+ */
+const action = async () => {
+  const appContext = await compileStrapi();
+  const app = await createStrapi(appContext).load();
+
+  const list = await app.admin.services.user.findPage({
+    select: ['id', 'firstname', 'lastname', 'email', 'isActive', 'blocked'],
+    populate: ['roles'],
+    pageSize: 25,
```

Comment:
Sadly the validation of this service doesn't allow for negative pageSize `-1` unless I swap to the db.query instead so either I need to change it to the db.query or we hard code the limit. I don't know which is best honestly